index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/936703f17d.js" crossorigin="anonymous"></script>
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <div class="header">
+            <div class="logo_menu">
+                <div class="logo"><img src="logo.png" alt="logo" width="30px" height="20px"></div>
+                <div class="burger"><i class="fa-solid fa-bars"></i></div>
+            </div>
+        </div>
+    </header>
+    <!--Intro section-->
+    <div class="intro">
```

Comment:
- [ ] It would be better to use a `<section>` tag instead of a `<div>` tag to wrap your headline content as it represents a generic standalone section of the document